bin/taskcat

```diff
@@ -57,15 +58,18 @@ def main():
                 raise
             else:
                 traceback.print_exc()
-        tcat_instance.stage_in_s3(taskcat_cfg)
-        tcat_instance.validate_template(taskcat_cfg, test_list)
-        tcat_instance.validate_parameters(taskcat_cfg, test_list)
-        # instance.stackcreate returns testdata object
-        testdata = tcat_instance.stackcreate(taskcat_cfg, test_list, args.stack_prefix)
-        tcat_instance.get_stackstatus(testdata, 5)
-        tcat_instance.createreport(testdata, 'index.html')
-        tcat_instance.cleanup(testdata, 5)
-
+        try:
+            tcat_instance.stage_in_s3(taskcat_cfg)
+            tcat_instance.validate_template(taskcat_cfg, test_list)
+            tcat_instance.validate_parameters(taskcat_cfg, test_list)
+            # instance.stackcreate returns testdata object
+            testdata = tcat_instance.stackcreate(taskcat_cfg, test_list, args.stack_prefix)
+            tcat_instance.get_stackstatus(testdata, 5)
+            tcat_instance.createreport(testdata, 'index.html')
+            tcat_instance.cleanup(testdata, 5)
+        except taskcat.exceptions.TaskCatException as e:
```

Comment:
catching all TaskCatException's here, then calling the raise1 function, should mimic the behavior of the previous sys.exit pattern